### DISCUSSION TITLE
Creating a character

### DISCUSSION THREAD

**Discussion by qbyss** [oobabooga/text-generation-webui#722] [Q&A] [answered]
I'm trying to save my character in cai_chat but I don't see a way to do that. There is the "Example" character but no way to export mine.  I can just save the conversation.

**Comment by qbyss** [comment]
I mean I know how to do it by editing file manually but was wondering if there was a feature in the webui to do that more naturally.

**Comment by bmoconno** [comment, marked as answer]
There's nothing built in yet, but there are some websites linked in the [wiki ](https://github.com/oobabooga/text-generation-webui/wiki)that are very good.

* [JSON character creator](https://oobabooga.github.io/character-creator.html)
* [AI Character Editor](https://zoltanai.github.io/character-editor/)
